Setup AnyCable

fargelus opened the pull request:
PR replaces ActionCable by AnyCable.
Works by default without `AnyCable::Rails::ConnectionFactory`.

fargelus committed:
Setup AnyCable

palkan commented:
> Works by default without AnyCable::Rails::ConnectionFactory.

Ha, okay. I've fixed that in master :) Please, rebase and try again. Also, I've updated the JS code to read URLs from meta tags (so no change in the JS code required anymore).

fargelus committed:
Connection factory used

fargelus commented:
> Ha, okay. I've fixed that in master :) Please, rebase and try again. Also, I've updated the JS code to read URLs from meta tags (so no change in the JS code required anymore).

Now connection factory is helpful :)
